src/test/mir-opt/simplify-locals-fixedpoint.rs

```diff
@@ -0,0 +1,15 @@
+// compile-flags: -Zmir-opt-level=1
```

Comment:
This is the default, right?

Reply:
It is for regular `rustc` invocations but for `mir-opt` tests, `-Zmir-opt-level=2` is the default.